### PR TITLE
Added how to build docs locally

### PR DESCRIPTION
**Pull request by jbusecke**
In #38 @iangrooms asked for instructions on how to build docs locally. I have added instructions here, but ironically I was not able to build them locally. I think there is a problem with my conda env (not resolving to the latest `myst_nb` version). 

@iangrooms could you try to follow these instructions and see if it works for you? Might just be some quirk of my local machine.

### CODE REVIEW

File: docs/how_to_contribute.rst [hunk 1]
```diff
@@ -61,3 +61,24 @@ until all checks pass.
 
 
 Once you got everything to pass, you can stage and commit your changes and push them to the remote github repository.
+
+How to change the documentation
+-------------------------------
+
+In order to build the documentation locally you should build and activate the docs environment::
+
+   conda env create -f docs/environment.yml
```
Comment (rabernat) [inline docs/how_to_contribute.rst:70]:
```suggestion
   mamba env create -f docs/environment.yml
```

We already recommend mamba elsewhere, why not here?

File: docs/how_to_contribute.rst [hunk 2]
```diff
@@ -61,3 +61,24 @@ until all checks pass.
...
+
+Then navigate to the docs folder and build the docs locally with::
+
+   cd docs
+
+   make html
+
+Once that is done you can open the created html files in `docs/_build/index.html` with your webbrowser::
+
+   open _build/index.html
```
Comment (rabernat) [inline docs/how_to_contribute.rst:82]:
```suggestion
   cd _build/html; python -m http.server
```

Better to preview the docs through an actual web server. [Sphinx autobuild](https://pypi.org/project/sphinx-autobuild/) is another good option.

Comment (iangrooms) [reply docs/how_to_contribute.rst:82]: Thanks! It works for me, though I do get this warning on the "make" command:

`WARNING: html_static_path entry '_static' does not exist`

Not sure what the `python -m http.server` command does. For me it appears to do nothing. But I can just open index.html with my browser instead.

File: docs/how_to_contribute.rst (hunk 1 above)
Comment (jbusecke) [reply docs/how_to_contribute.rst:70]: Sure thing.

File: docs/how_to_contribute.rst (hunk 2 above)
Comment (jbusecke) [reply docs/how_to_contribute.rst:82]: I am also just learning about this. @rabernat is there some documentation on this that I could read up on? Happy to change this though.

**Pull request merged**